Python/ceval.c

```diff
@@ -4741,14 +4739,14 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             goto start_frame;
         }
 
-        TARGET(CALL_FUNCTION_BUILTIN_O) {
+        TARGET(CALL_NO_KW_BUILTIN_O) {
```

Comment:
Note to self: no need to do the `extra_args` ceremony here because this opcode should never appear with `LOAD_METHOD`.